mobile/library/cc/engine_builder.h

```diff
@@ -64,8 +64,10 @@ class EngineBuilder {
   EngineBuilder& enableDrainPostDnsRefresh(bool drain_post_dns_refresh_on);
   EngineBuilder& enforceTrustChainVerification(bool trust_chain_verification_on);
   EngineBuilder& enablePlatformCertificatesValidation(bool platform_certificates_validation_on);
-  // Adds an RTDS layer to default config. Requires that ADS be configured
-  EngineBuilder& addRtdsLayer(const std::string& layer_name, int timeout_seconds = 5);
+  // Adds an RTDS layer to default config. Requires that ADS be configured.
+  EngineBuilder& addRtdsLayer(const std::string& layer_name, const int timeout_seconds = 5);
```

Comment:
since we are using 5 as the default timeout, we can make this a constant in the header file and use it in the default param values